Core/DisconnectMeRequest.swift

```diff
@@ -43,26 +43,26 @@ public class DisconnectMeRequest {
     
     private func handleResponse(response: Alamofire.DataResponse<Data>, completion: @escaping DisconnectMeRequestCompletion) {
         if let error = response.result.error {
-            complete(completion, withTrackers: nil, error: error)
+            complete(completion, count: 0, error: error)
             return
         }
         
         guard let data = response.result.value else {
-            complete(completion, withTrackers: nil, error: ApiRequestError.noData)
+            complete(completion, count: 0, error: ApiRequestError.noData)
             return
         }
-        
+
         do {
-            let trackers  = try self.parser.convert(fromJsonData: data)
-            complete(completion, withTrackers: trackers, error: nil)
+            let count = try DisconnectMeStore.shared.persist(data: data)
```

Comment:
Can we do the parsing and return that rather than storage?

Reply:
Agreed, will update for next PR.